cuckoo_time_translator_python/setup.py

```diff
@@ -6,7 +6,7 @@
 
 # fetch values from package.xml
 setup_args = generate_distutils_setup(
-    packages=['cuckoo_time_translator_python'],
+    packages=['cuckoo_time_translator_python', 'cuckoo_time_translator_python.timestamp_series'],
```

Comment:
That looks wrong to me
timestamp_series is a file so I dont think it should be in packages
Why do you think thats needed?

Reply:
Mistake. Thanks for spotting!